modules/account-lib/src/coin/baseCoin/blsKeyPair.ts

```diff
@@ -84,11 +87,25 @@ export abstract class BlsKeyPair implements BaseKeyPair {
    * @returns {boolean} is it valid?
    */
   public static isValidBLSPub(pub: string): boolean {
+    if (!initialized) {
+      throw new Error('BLS lib not yet initialized, please retry');
+    }
     try {
       BLS.PublicKey.fromHex(pub);
       return true;
     } catch (e) {
       return false;
     }
   }
+
+  public static aggregatePubkeys(pubKeys: Uint8Array[]): Buffer {
+    if (!initialized) {
+      throw new Error('BLS lib not yet initialized, please retry');
+    }
```

Comment:
might as well move this into a helper function `ensureInitialized` or something, just so its a 1-liner in all these fns instead of a 3-liner, and to ensure error msg consistency